app1/src/components/cards/UsersListCard.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+
+export default function MainCard({userTask}) {
+
+    return (
+      userTask.map(user => <Card sx={{ maxWidth: 400 }} key={userTask.id}>
```

Comment:
it is better to call it task inside map because you are looping tasks not users.